mod-audit-server/src/main/java/org/folio/rest/impl/AuditDataAcquisitionImpl.java

```diff
@@ -42,19 +46,16 @@ public void getAuditDataAcquisitionOrderById(String orderId, String sortBy, Audi
     LOGGER.debug("getAuditDataAcquisitionOrderById:: Retrieving Audit Data Acquisition Order By Id : {}", orderId);
     String tenantId = TenantTool.tenantId(okapiHeaders);
 
-    vertxContext.runOnContext(c -> {
-      try {
-        LOGGER.warn("Trying to get audit events by order id: {}", orderId);
-        orderAuditEventsService.getAuditEventsByOrderId(orderId, sortBy, sortOrder.name(), limit, offset, tenantId)
-          .map(GetAuditDataAcquisitionOrderByIdResponse::respond200WithApplicationJson)
-          .map(Response.class::cast)
-          .otherwise(this::mapExceptionToResponse)
-          .onComplete(asyncResultHandler);
-      } catch (Exception e) {
-        LOGGER.warn("Failed to get order audit events by order id: {}", orderId, e);
-        asyncResultHandler.handle(Future.succeededFuture(mapExceptionToResponse(e)));
-      }
-    });
+    try {
+      orderAuditEventsService.getAuditEventsByOrderId(orderId, sortBy, sortOrder.name(), limit, offset, tenantId)
+        .map(GetAuditDataAcquisitionOrderByIdResponse::respond200WithApplicationJson)
+        .map(Response.class::cast)
+        .otherwise(this::mapExceptionToResponse)
```

Comment:
interesting that mapExceptionToResponse use debug level instead of error, please change to error level and delete throwable.getMessage(), because this info already exists in throwable

Reply:
it is method that map exception to response, I think that is why debug level is used. In catch block I am using error level

Reply:
If for example Get audit data endpoint will throw an error due to DB failure, will we have error log in some place?
We will not catch an exception in catch block, we will have just failed future(promise)